ruleset/decoders/0005-wazuh_decoders.xml

```diff
@@ -55,45 +55,52 @@
 
 <decoder name="agent-buffer">
   <parent>wazuh</parent>
+  <type>wazuh</type>
   <prematch offset="after_parent">^Agent buffer:</prematch>
   <regex offset="after_prematch">^ '(\S+)'.</regex>
   <order>level</order>
 </decoder>
 
 <decoder name="agent-upgrade">
   <parent>wazuh</parent>
+  <type>wazuh</type>
   <prematch offset="after_parent">^Upgrade procedure |^Custom installation </prematch>
   <regex offset="after_prematch">on agent (\d\d\d)\s\((\S+)\):\s(\w+)</regex>
   <order>agent.id, agent.name, status</order>
 </decoder>
 
 <decoder name="agent-upgrade">
   <parent>wazuh</parent>
+  <type>wazuh</type>
   <regex>aborted:\s(\.+)$|failed:\s(\.+)$|lost:\s(\.+)$</regex>
   <order>error</order>
 </decoder>
 
 <decoder name="agent-upgrade">
   <parent>wazuh</parent>
+  <type>wazuh</type>
   <regex>started.\sCurrent\sversion:\sWazuh\s(\.+)$</regex>
   <order>agent.cur_version</order>
 </decoder>
 
 <decoder name="agent-upgrade">
   <parent>wazuh</parent>
+  <type>wazuh</type>
   <regex>succeeded.\sNew\sversion:\sWazuh\s(\.+)$</regex>
   <order>agent.new_version</order>
 </decoder>
 
 <decoder name="agent-restart">
   <parent>wazuh</parent>
+  <type>wazuh</type>
```

Comment:
We could add another test case for this decoder, like
```
# Wazuh Invalid Remote Configuration

log 1 pass = wazuh: Invalid remote configuration

rule = 220
alert = 7
decoder = wazuh
```

Reply:
Done! Thanks.